README.md

```diff
@@ -132,4 +132,4 @@ echo "JADE_UWU=true" >> ~/.zshrc <br>
 echo "JADE_UWU=true" >> ~/.bashrc <br>
 set -Ux JADE_UWU true <br>
 <br>
-if you want to have your log and crash output be "cute"
+if you want to have your log and crash output be “cute”
```

Comment:
what exactly is the change here? I don't see any difference between the quotes

Reply:
Nevermind, I'm looking at this from a different device now and it's properly rendering